ontology/yaml/resources/HVAC/entity_types/AHU.yaml

```diff
@@ -18,6 +18,24 @@
 
 ### Single Zone Units ###
 
+AHU_HWSC_ETM_SFM_FDPSM_SFVSC_EFVSC_SDM_EDM_CPC_WDPM_VSC:
```

Comment:
@Ciaran-Oflannagain even if the Economizer is fixed, we still must model it here -- DBO attempts to model things as they operate in real life, not as they digitally represent. I would imagine you still have data for a mixed_air_temperature_sensor or and outside_air_damper_percentage_command. Please use ECON and make necessary fields MISSING in your building config